app/controllers/api/v1/users_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
     user = User.find(params[:id])
     render json: UserSerializer.new(user), status: 200
   end
-
+# POST /api/v1/users
```

Comment:
Added this in for FE and talked through it with them so they could see a little better what was going on.

Reply:
Cool, that looks helpful